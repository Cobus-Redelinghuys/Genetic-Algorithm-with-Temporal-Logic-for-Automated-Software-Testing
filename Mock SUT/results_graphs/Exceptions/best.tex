Table \ref{tab:HP:GA:Exceptions:best fitness} tabulates the final average, standard deviations, and overall average for the best fitness that the different GATLAST configurations configured with different exceptions weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
exceptions & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 0.501 & 0.027 & 0.47\\\hline
0.25 & 0.494 & 0.045 & 0.452\\\hline
0.5 & 0.468 & 0.026 & 0.419\\\hline
0.75 & 0.481 & 0.021 & 0.435\\\hline
1.0 & 0.456 & 0.031 & 0.423\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the best fitness metric that the of different exceptions configured GATLASTs were achieved}
\label{tab:HP:GA:Exceptions:best fitness}
\end{table}