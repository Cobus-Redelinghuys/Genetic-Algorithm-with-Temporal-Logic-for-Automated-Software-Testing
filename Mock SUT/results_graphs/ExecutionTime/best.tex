Table \ref{tab:HP:GA:ExecutionTime:best fitness} tabulates the final average, standard deviations, and overall average for the best fitness that the different GATLAST configurations configured with different execution time weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
execution time & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 0.521 & 0.022 & 0.469\\\hline
0.25 & 0.481 & 0.032 & 0.45\\\hline
0.5 & 0.468 & 0.026 & 0.419\\\hline
0.75 & 0.456 & 0.025 & 0.431\\\hline
1.0 & 0.449 & 0.021 & 0.421\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the best fitness metric that the of different execution time configured GATLASTs were achieved}
\label{tab:HP:GA:ExecutionTime:best fitness}
\end{table}