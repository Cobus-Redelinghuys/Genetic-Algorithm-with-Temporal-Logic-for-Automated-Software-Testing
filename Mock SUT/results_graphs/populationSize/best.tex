Table \ref{tab:HP:GA:populationSize:best fitness} tabulates the final average, standard deviations, and overall average for the best fitness that the different GATLAST configurations configured with different population size weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
population size & Final average & Final standard deviation & Overall average\\
\hline
25 & 0.546 & 0.056 & 0.481\\\hline
50 & 0.468 & 0.026 & 0.419\\\hline
75 & 0.443 & 0.028 & 0.413\\\hline
100 & 0.421 & 0.031 & 0.391\\\hline
125 & 0.429 & 0.023 & 0.393\\\hline
150 & 0.415 & 0.034 & 0.389\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the best fitness metric that the of different population size configured GATLASTs were achieved}
\label{tab:HP:GA:populationSize:best fitness}
\end{table}