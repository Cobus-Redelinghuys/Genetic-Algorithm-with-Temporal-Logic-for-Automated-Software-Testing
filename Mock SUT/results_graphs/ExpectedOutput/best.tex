Table \ref{tab:HP:GA:ExpectedOutput:best fitness} tabulates the final average, standard deviations, and overall average for the best fitness that the different GATLAST configurations configured with different expected output weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
expected output & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 0.474 & 0.02 & 0.429\\\hline
0.25 & 0.48 & 0.025 & 0.434\\\hline
0.5 & 0.468 & 0.026 & 0.419\\\hline
0.75 & 0.49 & 0.021 & 0.454\\\hline
1.0 & 0.5 & 0.022 & 0.457\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the best fitness metric that the of different expected output configured GATLASTs were achieved}
\label{tab:HP:GA:ExpectedOutput:best fitness}
\end{table}